MonoGame.Framework/Audio/SoundEffect.OpenAL.cs

```diff
@@ -48,29 +48,9 @@ private void PlatformLoadAudioStream(Stream s)
             int freq;
 
             var stream = s;
-#if ANDROID
-            var needsDispose = false;
-            try
-            {
-                // If seek is not supported (usually an indicator of a stream opened into the AssetManager), then copy
-                // into a temporary MemoryStream.
-                if (!s.CanSeek)
-                {
-                    needsDispose = true;
-                    stream = new MemoryStream();
-                    s.CopyTo(stream);
-                    stream.Position = 0;
```

Comment:
@KonajuGames I'm sure we had this on Android for a reason. 
@mrhelmut did you test that this did not break android sound support?